ui/app/hooks/useTransactionDisplayData.js

```diff
@@ -111,7 +110,7 @@ export function useTransactionDisplayData (transactionGroup) {
     subtitleContainsOrigin = true
   } else if (transactionCategory === DEPLOY_CONTRACT_ACTION_KEY || transactionCategory === CONTRACT_INTERACTION_KEY) {
     category = TRANSACTION_CATEGORY_INTERACTION
-    title = (methodData?.name && camelCaseToCapitalize(methodData.name)) || (actionKey && t(actionKey)) || ''
+    title = (methodData?.name && camelCaseToCapitalize(methodData.name)) || t(transactionCategory) || ''
```

Comment:
If `transactionCategory` is falsy, will the return value of `t` also be falsy? Trying to understand how this will hit the `''` case.

Reply:
It should never hit this case. It can probably be safely removed. I'll update the PR